src/pages/_app.page.tsx

```diff
@@ -1,22 +1,29 @@
 import { Layout, Modal, Space } from 'antd'
 import { Content } from 'antd/lib/layout/layout'
-
-import { NetworkContext } from 'contexts/networkContext'
-import { NetworkName } from 'models/network-name'
-import { useContext, useLayoutEffect, useState } from 'react'
-
+import { useRouter } from 'next/router'
 import Navbar from 'components/Navbar'
-
+import { NetworkContext } from 'contexts/networkContext'
 import useMobile from 'hooks/Mobile'
+import { NetworkName } from 'models/network-name'
+import type { AppProps } from 'next/app'
+import React, { useContext, useEffect, useState } from 'react'
+import { CoreAppWrapper, Head } from 'components/common'
 
 import { readNetwork } from 'constants/networks'
 
-import Router from './Router'
+import '../styles/antd.css'
+import '../styles/index.scss'
 
-function App() {
+// TODO: Move this to each page where needed.
+const AppWrapper: React.FC = ({ children }) => {
   const [switchNetworkModalVisible, setSwitchNetworkModalVisible] =
     useState<boolean>()
 
+  const router = useRouter()
+  if (router.asPath.match(/^\/#\//)) {
```

Comment:
wdyt about solving this as a Rewrite in next.config.js? https://nextjs.org/docs/api-reference/next.config.js/rewrites#regex-path-matching

Reply:
Would love to do it, but unfortunately hash fragments are purely local - the server never sees it :(.

I actually exploited this fact at my last job to pass sensitive information via a link (which is kinda dumb, but client wanted it) so it doesn't go back to the server

Reply:
ohh makes sense, great to know 👍